.sdkmanrc

```diff
@@ -0,0 +1,2 @@
+java=11.0.23-tem
+maven=3.9.6
```

Comment:
please use 3.8.8 instead

```
$ ./mvnw --version
Apache Maven 3.8.8 (4c87b05d9aedce574290d1acc98575ed5eb6cd39)
Maven home: C:\Users\wing3\.m2\wrapper\dists\apache-maven-3.8.8-bin\67c30f74\apache-maven-3.8.8
Java version: 17.0.10, vendor: Eclipse Adoptium, runtime: C:\Program Files\Eclipse Adoptium\jdk-17.0.10.7-hotspot
Default locale: en_HK, platform encoding: Cp1252
OS name: "windows 11", version: "10.0", arch: "amd64", family: "windows"

```

Reply:
Done via 5196d35324654c096262c14e27a6bd9a94e3e7b3

I also updated the Version in README.md accordingly as i concluded from your review that this is the common baseline for the core team then.